composer.json

```diff
@@ -19,7 +19,8 @@
         "ext-tokenizer": "*",
         "composer/semver": "^3.2",
         "composer/xdebug-handler": "^3.0.3",
-        "doctrine/annotations": "^1.13",
+        "doctrine/annotations": "^1.13 || ^2",
```

Comment:
would the current CI setup have a job that run against v1, and another job that runs against v2 ?
also, can we support only v2?

Reply:
> would the current CI setup have a job that run against v1, and another job that runs against v2 ?

Yes, your low-deps job runs with Annotations 1.13, all the other jobs run with 2.0.

> also, can we support only v2?

You certainly can. However, many projects have Doctrine Annotations installed as a transitive dependency. If they install CS Fixer via `require-dev`, dropping support for v1 right away might cut them off from new CS Fixer releases until all of their dependencies are compatible with v2.

Supporting both major branches means hardly any extra effort on your side and you'd hardly gain anything from dropping v1. My suggesting would be to revisit this dependency in a couple of months and drop v1 once the adoption of v2 is higher.